plugins/modules/aci_interface_policy_leaf_fc_policy_group.py

```diff
@@ -0,0 +1,361 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Anvitha Jain <anvjain@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_interface_policy_leaf_fc_policy_group
+short_description: Manage fabric interface policy fiber policy groups (infra:AccBndlGrp, infra:AccPortGrp)
+description:
+- Manage fabric interface policy fiber policy groups on Cisco ACI fabrics.
+options:
+  policy_group:
+    description:
+    - Name of the fiber policy group to be added/deleted.
+    type: str
+    aliases: [ name, policy_group_name ]
+  description:
+    description:
+    - Description for the fiber policy group to be created.
```

Comment:
Do we want to introduce another way of documenting with sentences that end with `to be created`?

Reply:
rephrased.